.github/workflows/main.yaml

```diff
@@ -103,18 +103,13 @@ jobs:
         shell: cmd
         run: |
           choco install llvm
-        # run: |
-        #   curl --proto '=https' --tlsv1.2 -sSf https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe -L -o llvm-installer.exe
-        #   7z x llvm-installer.exe -oC:/llvm-10
-        #   echo C:/llvm-10/bin >> $GITHUB_PATH
-        #   echo "LLVM_SYS_100_PREFIX=C:/llvm-10" >> $GITHUB_ENV
-        #   echo "LIBCLANG_PATH=C:/llvm-10/bin/libclang.dll" >> $GITHUB_ENV
-      - name: Install LLVM (Unix)
-        if: matrix.os != 'windows-latest' && matrix.target != 'aarch64-apple-darwin'
+      - name: Install LLVM
+        if: matrix.os != 'windows-latest' && matrix.os != 'macos-11.0'
```

Comment:
`matrix.os != 'macos-11.0'` this will always be true, the name in `matrix.os` for macos is `macos-latest`

Reply:
What we want here is skip `macos-arm64` for which `matrix.os` is `macos-11.0`

Reply:
Oh, didn't see that! it was hidden in the diff viewer, looks good to me then